lib/connect-backbone-to-react.js

```diff
@@ -93,6 +93,14 @@ module.exports = function connectBackboneToReact(
       }
 
       createNewProps() {
+        // Bail out if our component has been unmounted.
+        // The only case where this flag is encountered is when this component
+        // is unmounted within an event handler but the 'all' event is still triggered.
+        // It is covered in a test case.
+        if (this.hasBeenUnmounted) {
```

Comment:
I'm following general react conventions where otherwise private variables are not prefixed at all. Cuz otherwise everything will ultimately be private and it's just more keystrokes.